tasks/main.yml

```diff
@@ -20,7 +20,8 @@
 
 - include_tasks: sysctl-setup.yml
 
-- include_tasks: kubelet-setup.yml
+- include_tasks: kubelet-setup.yml                # deprecated
```

Comment:
Just two spaces before the `# deprecated`

Reply:
hm. I first had less, but yaml-lint meant that it would like to have more. Maybe I had just one :)